inference-engine/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/strided_slice.cpp

```diff
@@ -11,85 +11,85 @@ using namespace LayerTestsDefinitions;
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {0, 0, 0, 0},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 4, 3 }, { 0, 0, 0, 0 }, { 2, 2, 4, 3 }, { 1, 1, 2, 1 },
-                                {1, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 4, 2 }, { 1, 0, 0, 1 }, { 2, 2, 4, 2 }, { 1, 1, 2, 1 },
-                                {0, 1, 1, 0}, {1, 1, 0, 0},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 1, 2, 4, 2 }, { 1, 0, 0, 0 }, { 1, 2, 4, 2 }, { 1, 1, -2, -1 },
-                                {1, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 4, 2 }, { 1, 0, 0, 0 }, { 1, 2, 4, 2 }, { 1, 1, -2, -1 },
-                                {0, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 0, 0, 0, 0 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {1, 1, 1, 1}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::I64, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {1, 1, 1, 1},  {},  {},  {},
-                                InferenceEngine::Precision::I64, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 2, 2, 2 }, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 },
-                                {0, 0, 0, 0}, {0, 0, 0, 0},  {},  {},  {},
-                                InferenceEngine::Precision::I64, CommonTestUtils::DEVICE_GPU),
-        stridedSliceParamsTuple({ 2, 3, 4, 5, 6 }, { 0, 1, 0, 0, 0 }, { 2, 3, 4, 5, 6 }, { 1, 1, 1, 1, 1 },
-                                {1, 0, 1, 1, 1}, {1, 0, 1, 1, 1},  {},  {0, 1, 0, 0, 0},  {},
-                                InferenceEngine::Precision::FP32, CommonTestUtils::DEVICE_GPU),
+std::vector<Parameters> ss_only_test_cases = {
+        Parameters{ { { 128, 1 }, { 0, 0, 0 }, { 0, 0, 0 }, { 1, 1, 1 },
```

Comment:
use `StrideSliceParams` instead of `Params`

Reply:
Thanks! Fixed.